data/i18n/hydrogen_fr.ts

```diff
@@ -1674,6 +1666,46 @@ Are you sure you want to save?</source>
         <source>Could not write to temporary directory %1.</source>
         <translation>Impossible d&apos;écrire dans le répertoire temporaire %1.</translation>
     </message>
+    <message>
+        <source>Replace with &amp;new Song</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <source>&amp;Import into Session</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <source>Import &amp;recent into Session</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <source>Export from Session &amp;as...</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <source>&amp;New</source>
+        <translation>&amp;Nouveau</translation>
+    </message>
+    <message>
+        <source>&amp;Open</source>
+        <translation>&amp;Ouvrir...</translation>
+    </message>
+    <message>
+        <source>Open &amp;recent</source>
+        <translation>Ouvrir un fichier ré&amp;cent...</translation>
+    </message>
+    <message>
+        <source>Save &amp;as...</source>
+        <translation>Sauveg&amp;arder sous...</translation>
+    </message>
+    <message>
+        <source>Song exported as: </source>
+        <translation>Morceau sauvegardé sous.</translation>
```

Comment:
The French translation should be : "`Morceau exporté sous : `".
Can you change it here?
Or you can merge it as is and I'll commit a fix afterwards. Fine with me too.

Reply:
Sure. I'll fix it